EssentialFeed/EssentialFeed/Image Comments/Presentation/ImageComment.swift

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public struct ImageComment: Equatable {
+	public let id: UUID
+	public let message: String
+	public let created_at: Date
```

Comment:
The convention in this codebase is camelcase. Please, rename the property:

```suggestion
	public let createdAt: Date
```